src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -911,6 +911,10 @@
 examples:
   - name: Get the details of an SSL certificate.
     text: az network application-gateway ssl-cert show -g MyResourceGroup --gateway-name MyAppGateway -n MySslCert
+  - name: Display the expiry date of SSL certificate. The certificate is returned in PKCS7 format from which the expiration date needs to be retrieved.
+    text: |
+        publiccert=`az network application-gateway ssl-cert show -g MyResourceGroup --gateway-name MyAppGateway --name mywebsite.com --query publicCertData -o tsv`
+        echo -e "-----BEGIN CERTIFICATE-----\n$publiccert\n-----END CERTIFICATE-----" | fold -w 64 | openssl pkcs7 -print_certs | openssl x509 -noout -enddate
```

Comment:
```suggestion
        az network application-gateway ssl-cert show -g MyResourceGroup --gateway-name MyAppGateway --name mywebsite.com --query publicCertData -o tsv | fold -w 64 | openssl pkcs7 -print_certs | openssl x509 -noout -enddate
```

Shall we make it one line?